--- origsrc/cinepaint-0.22-1/configure.in	2007-06-12 00:51:17.000000000 -0500
+++ src/cinepaint-0.22-1/configure.in	2008-04-07 08:56:43.171875000 -0500
@@ -420,11 +420,11 @@
   if test -z "$XMU_LIB"; then
     AC_CHECK_LIB(Xmu, XmuClientWindow,
       [AC_CHECK_HEADER(X11/Xmu/WinUtil.h,
-    CINEPAINT_REMOTE="cinepaint-remote"; XMU_LIB='-lXmu -lXt -lSM -lICE',
+    CINEPAINT_REMOTE='cinepaint-remote$(EXEEXT)'; XMU_LIB='-lXmu -lXt -lSM -lICE',
     [AC_MSG_WARN(*** cinepaint-remote will not be built (XMU header file not found) ***)])],
     [AC_CHECK_LIB(Xmu, XmuUpdateMapHints,
       [AC_CHECK_HEADER(X11/Xmu/WinUtil.h,
-        CINEPAINT_REMOTE="cinepaint-remote"; XMU_LIB='-lXmu -lXt',
+        CINEPAINT_REMOTE='cinepaint-remote$(EXEEXT)'; XMU_LIB='-lXmu -lXt',
         [AC_MSG_WARN(*** cinepaint-remote will not be built (XMU header file not found) ***)])],
     [AC_MSG_WARN(*** cinepaint-remote will not be built (XMU library not found) ***)], -lXt)], -lXt -lSM -lICE)
   fi
--- origsrc/cinepaint-0.22-1/lib/Makefile.am	2007-05-10 01:59:09.000000000 -0500
+++ src/cinepaint-0.22-1/lib/Makefile.am	2008-04-06 16:15:07.375000000 -0500
@@ -93,7 +93,7 @@
 
 ## libcinepaint_la_LDFLAGS = -version-info $(MAJOR_VERSION):$(MINOR_VERSION):$(MICRO_VERSION)
 
-libcinepaint_la_LDFLAGS = -version-info $(MAJOR_VERSION):$(MINOR_VERSION):0
+libcinepaint_la_LDFLAGS = -version-info $(MAJOR_VERSION):$(MINOR_VERSION):0 -no-undefined
 
 
 INCLUDES = $(X_CFLAGS) -I$(top_srcdir)	-I$(includedir)	-I../wire -I../fl_i18n
--- origsrc/cinepaint-0.22-1/lib/fl_i18n/Makefile.am	2007-03-30 04:45:22.000000000 -0500
+++ src/cinepaint-0.22-1/lib/fl_i18n/Makefile.am	2008-04-07 00:04:30.093750000 -0500
@@ -19,8 +19,7 @@
 library_includedir=${pkgincludedir}
 #library_include_HEADERS = fl_i18n.H
 
-libcinepaint_fl_i18n_la_LDFLAGS = -version-info $(MAJOR_VERSION):$(MINOR_VERSION):0
-
+libcinepaint_fl_i18n_la_LDFLAGS = -version-info $(MAJOR_VERSION):$(MINOR_VERSION):0 -no-undefined `$(FLTK_CONFIG) --use-images --ldflags`
 
 INCLUDES = $(USE_THREADS) $(FLTK_CFLAGS) $(X_CFLAGS) \
 	-I$(top_srcdir) -I$(includedir)
--- origsrc/cinepaint-0.22-1/libhalf/Makefile.am	2007-01-13 11:22:35.000000000 -0600
+++ src/cinepaint-0.22-1/libhalf/Makefile.am	2008-04-06 16:16:34.609375000 -0500
@@ -8,7 +8,7 @@
 	cinepaint_half.cpp cinepaint_half.h \
 	halfFunction.h halfLimits.h
 
-libcinepaintHalf_la_LDFLAGS = -version-info $(MAJOR_VERSION):$(MINOR_VERSION):0
+libcinepaintHalf_la_LDFLAGS = -version-info $(MAJOR_VERSION):$(MINOR_VERSION):0 -no-undefined
 #libHalf_la_LDFLAGS = -version-info @LIBTOOL_VERSION@ -no-undefined
 
 #libHalfincludedir = $(includedir)/OpenEXR
@@ -25,10 +25,10 @@
 
 toFloat_SOURCES = toFloat.cpp
 
-eLut.h:	eLut
+eLut.h:	eLut$(EXEEXT)
 	./eLut > eLut.h
 
-toFloat.h: toFloat
+toFloat.h: toFloat$(EXEEXT)
 	   ./toFloat > toFloat.h
 
 BUILT_SOURCES = eLut.h toFloat.h
--- origsrc/cinepaint-0.22-1/plug-ins/bracketing_to_hdr/Makefile.am	2006-12-29 09:45:47.000000000 -0600
+++ src/cinepaint-0.22-1/plug-ins/bracketing_to_hdr/Makefile.am	2008-04-07 00:29:16.078125000 -0500
@@ -45,11 +45,11 @@
 	-I$(includedir)
 
 LDADD = \
-	$(X_LIBS) \
-	$(top_builddir)/lib/libcinepaint.la \
 	./br_core/libbr_core.la \
 	./FL_adds/libfl_adds.la \
 	./gui/libgui.la \
+	$(X_LIBS) \
+	$(top_builddir)/lib/libcinepaint.la \
 	$(FLTK_LIBS) \
 	-lc
 
